e2e/tests/params_keys_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build e2e
```

Comment:
I think we should rename this file to `keys_utils_test.go` or `keys_helpers_test.go` because it extends beyond params.

Reply:
Good call out. I went with `keyring_helpers_test.go.